snuba/util.py

```diff
@@ -183,14 +140,17 @@ def escape_literal(
         raise ValueError("Do not know how to escape {} for SQL".format(type(value)))
 
 
-def time_request(name):
-    def decorator(func):
+F = TypeVar("F", bound=Callable[..., Any])
+
+
+def time_request(name: str) -> Callable[[F], F]:
+    def decorator(func: F) -> F:
         @wraps(func)
-        def wrapper(*args, **kwargs):
+        def wrapper(*args: Any, **kwargs: Any) -> Any:
```

Comment:
I think there should be a better way to do this. Will look again later.

Reply:
Seems fine to me in this case for these all to be Any